sdks/java/core/src/test/java/org/apache/beam/sdk/coders/ListCoderTest.java

```diff
@@ -128,12 +128,19 @@ public void testNotConsistentWithEquals() {
   @Test
   public void testConsistentWithEquals() {
     ListCoder<Integer> coder = ListCoder.of(VarIntCoder.of());
-    assertTrue(coder.consistentWithEquals());
+    assertFalse(coder.consistentWithEquals());
```

Comment:
Let's just delete this test, and the other ones. Once `structuralValue` is defined, the return value from `consistentWithEquals` does not matter. In fact, the coder _is_ consistent with equals, but we don't care.

Reply:
removed